SemanticExtraSpecialProperties.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-use SESP\HookRegistry;
+use SMW\MediaWiki\Hooks\HookRegistry;
 
```

Comment:
This is not correct, we want [0] and as you can see the tests fails.

[0] https://github.com/SemanticMediaWiki/SemanticExtraSpecialProperties/blob/master/src/HookRegistry.php